src/Client.php

```diff
@@ -67,7 +67,7 @@ public function __construct(LoopInterface $loop = null, $url = null)
             // send everything through a local unix domain socket
             $connector = new FixedUriConnector(
                 $url,
-                new UnixConnector($loop)
+                new UnixConnector()
```

Comment:
Argument should not be omitted if explicit loop is given.

Reply:
:+1: